integration-test/background/grades.js

```diff
@@ -5,7 +5,7 @@ const tests = [
     { url: 'duckduckgo.com', siteGrade: 'A', enhancedGrade: 'A' },
     { url: 'theguardian.com', siteGrade: 'D-', enhancedGrade: 'B+' },
     { url: 'google.com', siteGrade: 'D', enhancedGrade: 'D' },
-    { url: 'amazon.com', siteGrade: ['D-', 'C+'], enhancedGrade: 'C+' },
```

Comment:
We allow enhanced grade (C) to be < than the default grade (C+)? 🤔

Reply:
Good catch. I'll try removing C+ and see how it goes.